src/devices/Button/ButtonBase.cs

```diff
@@ -19,6 +19,8 @@ public class ButtonBase : IDisposable
 
         private long _doublePressTicks;
         private long _holdingMs;
+        private TimeSpan _debounceTime;
+        private long _singlePressTicks;
```

Comment:
this is misnamed - currently `_doublePressTicks` means amount of time between presses while `_singlePressTicks` means last time we pressed the button. I'd recommend renaming it to something like `_lastPressedTicks` or `_debounceStart` or something along the lines - as is makes reading the code a bit confusing

Reply:
I'll go with `_debounceStartTicks`